blueprint/core/blueprint_EcmascriptEngine.cpp

```diff
@@ -151,6 +151,12 @@ namespace blueprint
             reset();
         }
 
+        ~Pimpl()
+        {
+            stopTimer();
```

Comment:
Found some minor funk in shutdown...

Reply:
Ah, wonder what you were seeing here? I found something similar and just pushed a fix on the latest master

Reply:
I think what happened was that the timer message kept up for 1 frame too late, and dereferenced the context pointer when it was invalid.